### PR TITLE
Main page

### PR DESCRIPTION
**Pull request by Anders-Bacon**
Currently i am working on the Carousel of the frontpage and making that work in a hard coded format. 
The same can be said for the category part of the page, the component has been made and can just be copied, but isnt really styled or anything yet

### CODE REVIEW

**Comment by Anders-Bacon** [comment]
Currently the only thing the frontpage needs is a connection to the database so that the right content can be displayed

**Comment by Anders-Bacon** [comment]
Main page is done now, with both my indivdual extension and other things. Everything now has php variables and uses the data from the database

**Review by MidnightRocket — CHANGES_REQUESTED** [review]
Separate individual from group work before merge can happen

File: app/Http/Controllers/categoryController.php
Comment (MidnightRocket) [inline app/Http/Controllers/categoryController.php:1]: Please rename to UpperCamel case

File: routes/web.php [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 use Illuminate\Support\Facades\Route;
 
+use App\Http\Controllers\CarouselController;
+use App\Http\Controllers\categoryController;
```
Comment (MidnightRocket) [inline routes/web.php]: And rename here please

**Review by MidnightRocket — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/categoryController.php
Comment (Anders-Bacon) [reply app/Http/Controllers/categoryController.php:1]: Done

File: routes/web.php (hunk 1 above)
Comment (Anders-Bacon) [reply routes/web.php]: done

File: resources/views/components/layouts/base.blade.php [hunk 2]
```diff
@@ -1,11 +1,15 @@
 <x-layouts.minimum {{ $attributes }}>
     <x-slot:head>
+        <link rel="stylesheet" href="{{asset('css/app.css')}}">
```
Comment (MidnightRocket) [inline resources/views/components/layouts/base.blade.php]: This line can be removed

File: app/Http/Controllers/categoryController.php
Comment (MidnightRocket) [inline app/Http/Controllers/categoryController.php:1]: Should be renamed to CamelCase

**Review by MidnightRocket — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/categoryController.php
Comment (Anders-Bacon) [reply app/Http/Controllers/categoryController.php:1]: Done

**Review by MidnightRocket — APPROVED** [review]
Looks good now

**Pull request merged**